### PR TITLE
replace FSA with Array2 in RmEpsilons

### PR DESCRIPTION
**Pull request by qindazhu**

### CODE REVIEW

File: k2/csrc/rmepsilon_test.cc [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+#include "k2/csrc/fsa_util.h"
+#include "k2/csrc/properties.h"
+
+namespace k2 {
+class RmEpsilonTest : public ::testing::Test {
+ protected:
+  RmEpsilonTest() {
+    std::vector<Arc> arcs = {
+        {0, 4, 1},  {0, 1, 1},  {1, 2, 0},  {1, 3, 0},  {1, 4, 0},
+        {2, 7, 0},  {3, 7, 0},  {4, 6, 1},  {4, 6, 0},  {4, 8, 1},
```
Comment (csukuangfj) [inline k2/csrc/rmepsilon_test.cc:28]: 0 -> kEpsilon

**Comment by csukuangfj** [comment]
+1

File: k2/csrc/rmepsilon_test.cc (hunk 1 above)
Comment (qindazhu) [reply k2/csrc/rmepsilon_test.cc]: Updated, thanks.

File: k2/csrc/rmepsilon_test.cc [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#include "k2/csrc/fsa_util.h"
+#include "k2/csrc/properties.h"
+
+namespace k2 {
+class RmEpsilonTest : public ::testing::Test {
+ protected:
+  RmEpsilonTest() {
+    std::vector<Arc> arcs = {
+        {0, 4, 1},        {0, 1, 1},        {1, 2, kEpsilon}, {1, 3, kEpsilon},
+        {1, 4, kEpsilon}, {2, 7, kEpsilon}, {3, 7, kEpsilon}, {4, 6, 1},
```
Comment (danpovey) [inline k2/csrc/rmepsilon_test.cc:28]: FYI, I would have been fine with zeros here.  Either way.

Comment (qindazhu) [reply k2/csrc/rmepsilon_test.cc:28]: Agree with you  as it's just test code. Anyway, will not change back to zeros as either zeros or kEpsilon would be fine.

**Comment by qindazhu** [comment]
Merging

**Pull request merged**